views/apply-success.njk

```diff
@@ -0,0 +1,26 @@
+{% extends "./layouts/main.njk" %}
+
+{% block pageTitle %}
+  Successful Application
+{% endblock %}
+
+{% block links %}
+<link rel="stylesheet" href="/css/style.css">
+{% endblock %}
+
+{% block content %}
+
+
+<div class="d-flex justify-content-center">
+  <div>
+
+{% if errormessage %}
```

Comment:
Why does this page display error messages at the same time as showing the success message

Reply:
Removed